Kitodo/src/test/java/org/kitodo/services/data/BatchServiceIT.java

```diff
@@ -36,8 +37,9 @@ public static void prepareDatabase() throws DAOException, IOException, ResponseE
     }
 
     @AfterClass
-    public static void cleanDatabase() {
+    public static void cleanDatabaseAndIndex() throws IOException, ResponseException{
```

Comment:
It is better to introduce new functionality if this functionality is used. Prepared functionality got sometimes lost and a future review everybody could not explain this. I think it is better remove this and re-add this if this is needed.